ibis/expr/types/relations.py

```diff
@@ -4677,6 +4677,13 @@ def value_counts(self) -> ir.Table:
             lambda t: t.count().name("_".join(columns) + "_count")
         )
 
+    def unnest(self, column, offset: str | None = None) -> Table:
```

Comment:
+1 This seems pretty natural to me. :-)

Reply:
Would we want to call it explode to mimic pandas? That said, I suppose this makes sense with regards to internal consistency with the other "unnest".

Reply:
One thing that I think might be a bit confusing is that sqlglot calls unnest-in-select-position `explode` and unnest-in-join-position `unnest` so if called the unnest here explode that would be the reverse convention of sqlglot.

Personally I think keeping the name the same, seems like it will be better for conceptual clarity, since the operations are doing the same thing at their core.